nextmv/nextmv/cloud/application.py

```diff
@@ -37,24 +38,57 @@ class DownloadURL(BaseModel):
 
 
 class PollingOptions(BaseModel):
-    """Options to use when polling for a run result."""
-
-    backoff: float = 1
-    """Backoff factor to use between polls. Leave this at 1 to poll at a
-    constant rate."""
-    delay: float = 1
-    """Delay to use between polls, in seconds."""
+    """
+    Options to use when polling for a run result.
+
+    The Cloud API will be polled for the result. The polling stops if:
+
+    * The maximum number of polls (tries) are exhausted. This is specified by
+      the `max_tries` parameter.
+    * The maximum duration of the polling strategy is reached. This is
+      specified by the `max_duration` parameter.
+
+    Before conducting the first poll, the `initial_delay` is used to sleep.
+    After each poll, a sleep duration is calculated using the following
+    strategy, based on exponential backoff with jitter:
+
+    ```
+    sleep_duration = min(`max_delay`, `delay` + `backoff` * 2 ** i + Uniform(0, `jitter`))
+    ```
+
+    Where:
+    * i is the retry (poll) number.
+    * Uniform is the uniform distribution.
+
+    Note that the sleep duration is capped by the `max_delay` parameter.
+    """
+
+    backoff: float = 0.9
+    """
+    Exponential backoff factor, in seconds, to use between polls. Leave this at
+    1 to poll at a constant rate.
+    """
+    delay: float = 0.1
+    """Base delay to use between polls, in seconds."""
     initial_delay: float = 1
-    """Initial delay to use before starting the polling strategy, in
-    seconds."""
+    """
+    Initial delay to use before starting the polling strategy, in seconds.
+    """
     max_delay: float = 20
-    """Maximum delay to use between polls, in seconds. This parameter is
-    activated when the backoff parameter is greater than 1, such that the delay
-    is increasing after each poll."""
+    """Maximum delay to use between polls, in seconds."""
     max_duration: float = 300
     """Maximum duration of the polling strategy, in seconds."""
-    max_tries: int = 20
+    max_tries: int = 100
     """Maximum number of tries to use."""
+    jitter: float = 1
+    """
+    Jitter to use for the polling strategy. A uniform distribution is sampled
+    between 0 and this number. The resulting random number is added to the
+    delay for each poll, adding a random noise. Leave this at 0 to avoid using
+    random jitter.
```

Comment:
```suggestion
    delay for each poll, adding a random noise. Set this to 0 to avoid using
    random jitter.
```

Just while I was at it, since it is technically not at 0 by default.